packages/shared/src/lib/featureManagement.ts

```diff
@@ -82,6 +82,10 @@ export const featureValidLanguages = new Feature('valid_languages', {
   ko: 'Korean',
 });
 
+export const featurePlusButtonColors = new Feature<
+  'avocado' | 'cabbage' | 'onion' | 'cheesebacon' | 'onionbacon' | ''
+>('plus_button_colors', '');
+
```

Comment:
Maybe we could implement a enum with those values?

Reply:
Yeah, enum or at least have these values in a `type` and use that `type` on the function where we do the switch-case.